addons/website_sale/views/templates.xml

```diff
@@ -1985,4 +1985,13 @@
             <t t-set="html_data" t-value="dict(html_data, **{'data-add2cart-redirect': website.cart_add_on_page and '1' or '0'})"/>
         </xpath>
     </template>
+
+    <template id="product_category_extra_link" name="Product Category Extra Link">
+        <button t-if="len(categories) == 1" class="btn btn-link btn-sm pr-0" disabled="disabled">Category:</button>
+        <button t-else="" class="btn btn-link btn-sm pr-0" disabled="disabled">Categories:</button>
```

Comment:
What about putting the condition on the text only so that all the rest is defined only once ?

Reply:
Done, thanks ! :)